### PR TITLE
Feinstaub migrations

### PR DESCRIPTION
**Pull request by karimkawambwa**
## Description

Track model changes made by opendata-stuttgart on feinstaub since they are not checking in the migration files along with any model changes they make. This will allow us to keep track of any changes and apply approved migrations on each release.

Fixes # (issue)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by karimkawambwa** [comment]
@kilemensi 👀

File: README.md [hunk 1]
```diff
@@ -48,6 +49,15 @@ Docker compose make commands:
...
 The Dockerfile is written for production since dokku is being used and it will look for Dockerfile.
 
+### Migrations
+
+Be sure to check in migrations with every model changes and be sure to review and test these changes.
+
+Open Stuttgart don't check in their migrations and so:
+
+- Install the latest pull from their repo; `pip install git+https://github.com/opendata-stuttgart/feinstaub-api`
+- Make sensorsafrica migrations and `feinstaub_migrations` with command; `python manage.py makemigrations`
```
Comment (kilemensi) [inline README.md]: Lets change this description here to something like:

> `opendata-stuttgart/feinstaub-api` prefer generating and applying migration to the database at the point of deployment (probably to reduce the number of changes to be applied). We, on the other hand, prefer the [Django recommended approach](https://docs.djangoproject.com/en/1.11/topics/migrations/#the-commands) of creating and review migration files at the development time, and then applying the same migration files to different environments; `dev`, `staging` and eventually `production`.

And this should probably be in *CONTRIBUTING* section and not *DEVELOPMENT*.

File: sensorsafrica/settings.py [hunk 2]
```diff
@@ -133,3 +133,8 @@
 # Simplified static file serving.
 # https://warehouse.python.org/project/whitenoise/
 STATICFILES_STORAGE = "whitenoise.storage.CompressedManifestStaticFilesStorage"
+
+# Put fenstaub migrations into sensorsafrica
+MIGRATION_MODULES = {
+    'sensors': 'sensorsafrica.openstuttgart.feinstaub.sensors.migrations'
```
Comment (kilemensi) [inline sensorsafrica/settings.py]: QQ: Is this for `sensors` or `feinstaub.sensors` app?

File: README.md [hunk 3]
```diff
@@ -14,6 +14,7 @@ To get the project up and running:
 - Use virtualenv to create your virtual environment; `virtualenv venv`
 - Activate the virtual environment; `source venv/bin/activate`
 - Install the requirements; `pip install .`
+- Install feinstaub; `pip install git+https://github.com/opendata-stuttgart/feinstaub-api`
```
Comment (kilemensi) [inline README.md]: NP: I think this should be _before_ installing requirements i.e. `sensors.AFRICA-api` _depends_ on it and not the other way round.

File: sensorsafrica/settings.py (hunk 2 above)
Comment (karimkawambwa) [reply sensorsafrica/settings.py]: its`feinstaub.sensors`

**Comment by karimkawambwa** [comment]
@kilemensi 👀 

From the slack conversation, is this approved?

File: README.md [hunk 4]
```diff
@@ -91,6 +92,13 @@ For more information read [Deploying to Dokku](http://dokku.viewdocs.io/dokku/de
 0 * * * * dokku enter sensorsafrica-staging web python3 manage.py calculate_data_statistics >> /var/log/cron.log 2>&1
 ```
 
+## Contributing
+
+opendata-stuttgart/feinstaub-api prefer generating and applying migration to the database at the point of deployment (probably to reduce the number of changes to be applied).
```
Comment (kilemensi) [inline README.md]: Make opendata-stuttgart/feinstaub-api a link to the repo or at least standout like `opendata-stuttgart/feinstaub-api`

File: README.md [hunk 5]
```diff
@@ -91,6 +92,13 @@ For more information read [Deploying to Dokku](http://dokku.viewdocs.io/dokku/de
...
+We, on the other hand, prefer the Django recommended approach of creating and reviewing migration files at the development time, and then applying the same migration files to different environments; dev, staging and eventually production.
+
+- Make sensorsafrica migrations and `feinstaub_migrations` with command; `python manage.py makemigrations`
```
Comment (kilemensi) [inline README.md]: Change this into a normal paragraph like (this is just an example):

Hence, with any contribution, include both `sensors.AFRICA-api` and `opendata-stuttgart/feinstaub-api` migration files by running `python manage.py makemigrations` command before creating a PR.

**Pull request merged**